src/i18n/locales/en.json

```diff
@@ -28,7 +31,7 @@
   "toolbar-button-upload-settings": "Open Upload Settings",
   "upload-notification": "Currently uploading",
   "upload-settings-modal-title": "Opencast Upload Settings",
-  "upload-settings-button-validate": "Validate Settings",
+  "upload-settings-button-store": "Store Settings",
```

Comment:
I think "save settings" is used more commonly than "store settings" and should therefore be preferred so that the UI feels more familiar? Not entirely sure though.